static/src/javascripts/projects/common/modules/identity/manage-account.js

```diff
@@ -16,6 +16,8 @@ import {
 } from './modules/switch';
 import { addUpdatingState, removeUpdatingState } from './modules/button';
 
+const ERR_HTML_PREF_NOT_FOUND = `Can't find HTML preference`;
```

Comment:
Would it make sense to prefix the error message with the subsystem (Identity) or make it a little bit less generic?

Reply:
just renamed it but also, is there a more generic/global error handling system at play? i'd love to tap into that instead of just leaving it there